Login/app/src/main/java/com/projetos/txoza/util/User.kt

```diff
@@ -9,7 +9,7 @@ class User(
    private val userPassword = password
     private val userEmail = email
 
-    fun validateUSer(email: String, password: String): Boolean {
+    fun validateUser(email: String, password: String): Boolean {
```

Comment:
This class user should be in a `model` package and in the `utils` you could create an `UserValidator` with a method called `validate` that receives an User and returns if the user is valid